packages/firestore/src/model/mutation.ts

```diff
@@ -302,17 +276,25 @@ export abstract class Mutation {
     localWriteTime: Timestamp
   ): MaybeDocument | null;
 
-  abstract isEqual(other: Mutation): boolean;
-
   /**
-   * If applicable, returns the field mask for this mutation. Fields that are
-   * not included in this field mask are not modified when this mutation is
-   * applied. Mutations that replace the entire document return 'null'.
+   * If applicable, returns the base value to persist with this mutation. If a
```

Comment:
When is this not `applicable`?

Reply:
Replaced with "If this mutation is not idempotent"